transaction-service-api/src/main/java/pl/adamboguszewski/transactionservice/service/api/transaction/CreateTransactionRequest.java

```diff
@@ -30,10 +30,10 @@
         UUID checkoutId;
 
         @NotNull
```

Comment:
We have @NotEmpty annotation for collections too, don't use it here for now but remember for future development.